test/unittests/parser_test.cpp

```diff
@@ -1118,12 +1118,13 @@ TEST(parser, code_section_unsupported_fp_instructions)
         }
         func_bin += bytes{0xb};  // end
         const auto code_bin = add_size_prefix(func_bin);
-        const auto section_contents = make_vec({code_bin});
-        const auto bin = bytes{wasm_prefix} + make_section(10, section_contents);
+        const auto code_section = make_vec({code_bin});
+        const auto function_section = make_vec({"00"_bytes});
+        const auto bin =
+            bytes{wasm_prefix} + make_section(3, function_section) + make_section(10, code_section);
 
-        const auto expected_msg =
-            std::string{"unsupported floating point instruction "} + std::to_string(instr);
-        EXPECT_THROW_MESSAGE(parse(bin), parser_error, expected_msg.c_str());
+        const auto module = parse(bin);
+        EXPECT_EQ(module.codesec.size(), 1);
```

Comment:
Maybe rename this test now, because it's kind of not "unsupported" now

Reply:
It is still unsupported in terms of execution, but yeah, supported in terms of parsing.

Reply:
Done.